Formula/ksync.rb

```diff
@@ -0,0 +1,27 @@
+class Ksync < Formula
+  desc "Sync files between your local system and a kubernetes cluster"
+  homepage "https://ksync.github.io/ksync/"
+  url "https://github.com/ksync/ksync.git",
+    :tag  => "0.4.5"
+  sha256 "bd47c5b5418bfc6d2f08abe855a52d4189f8bf380747cdd4e5c1a3ca0a87498a"
+  license "Apache-2.0"
+
```

Comment:
I get an audit error about this whitespace, but I see it in many other formulae? Is this incorrect?